.babelrc

```diff
@@ -0,0 +1,12 @@
+{
+    "presets": [
+      "@babel/preset-env",
+      "@babel/preset-react"
+    ],
+    "plugins": [
```

Comment:
this is really interesting. Why did you add these plugins along with your presets? What purpose are they serving in your code?